Samantha-samanthalpark/Homework-3/json-homework/json-hw.js

```diff
@@ -0,0 +1,45 @@
+
+/*
+1. Convert the string `jsonString` to a Javascript Object and store it in a variable
+called `flickrObj`
+*/
+
+let flickrObj = JSON.parse(jsonString);
+
+console.log(flickrObj);
+
+/*
+2. Iterate (this means 'use a loop') and do a console.log of each of the item titles
+with the date the photo was taken. The first console.log should return the following:
+
+"Edelweiss Air, Airbus A320-214, HB-IJU 2016-02-23T18:48:14Z"
+
+*/
+for(let i = 0; i < 20; i++){
```

Comment:
How could you change the code for your loop so it would always iterate over all of the array elements, even if one was added or removed down the line?